### DISCUSSION TITLE
🚀 Feature Request: set the incomming country in Wrangler Dev

### DISCUSSION THREAD

**Discussion by alex-h-strachan** [cloudflare/workers-sdk#13536] [Feature Requests]
### Describe the solution

Wrangler Dev exposes no interface to set the country of the request for the `worker.fetch()` api

We need to be able to test that our worker has different behaviors by the `request.cf.country` set in the request, but there's no hook in the SDK to achieve that.

**Comment by RamIdeas** [comment]
In local mode, you should be able to set the `cf` property in the RequentInit (2nd) param of `fetch`.

This might not be supported in unstable_dev because it doesn't use Miniflare's dispatchFetch function under-the-hood because unstable_dev also needs to support remote-mode.

Assigning this to myself: I will look into this and ensure it is supported by the work to improve wrangler's dev servers which is currently in-progress
